Adyen/UI/Form/Items/Address/AddressViewModel.swift

```diff
@@ -27,12 +33,12 @@ internal struct AddressViewModelBuilderContext {
     internal var isOptional: Bool
 }
 
-internal struct AddressViewModel {
+public struct AddressViewModel {
 
     internal var labels: [AddressField: LocalizationKey]
     internal var placeholder: [AddressField: LocalizationKey]
-    internal var optionalFields: [AddressField]
-    internal var schema: [FormScheme]
+    public var optionalFields: [AddressField]
+    public var schema: [FormScheme]
```

Comment:
`///:nodoc:` for both properties? :-)

also would be great to have them `private(set)` or at least `internal(set)`